B827EBFFFEA371DA

```diff
@@ -0,0 +1,18 @@
+{ 
```

Comment:
the file is missing the `.json` exception. please add it and then I'll merge. thanks!